modules/plugins/languages/markdown.nix

```
@@ -117,7 +117,13 @@ in {
             '';
           };
 
-        setupOpts = mkPluginSetupOption "render-markdown" {};
+        setupOpts = mkPluginSetupOption "render-markdown" {
+          file_types = lib.mkOption {
+            type = listOf (nullOr str);
+            default = [];
+            description = "List of buffer filetypes to enable this plugin in. This will cause the plugin to attach to new buffers who have any of these filetypes.";
```

```suggestion
            description = ''
            	List of buffer filetypes to enable this plugin in.
            	
            	This will cause the plugin to attach to new buffers who
            	have any of these filetypes.
            '';
```

Please make this a multi-line string. General rule of thumb is 80 lines max per string *excluding* the indent.